src/workerd/api/sql.c++

```diff
@@ -60,6 +97,45 @@ bool SqlStorage::allowTransactions() const {
   return false;
 }
 
+jsg::JsValue SqlStorage::wrapSqlValue(jsg::Lock& js, SqlValue value) {
+  KJ_IF_SOME(v, value) {
+    KJ_SWITCH_ONEOF(v) {
+      KJ_CASE_ONEOF(bytes, kj::Array<byte>) {
+        return jsg::JsValue(js.wrapBytes(kj::mv(bytes)));
+      }
+      KJ_CASE_ONEOF(text, kj::StringPtr) {
+        return js.str(text);
+      }
+      KJ_CASE_ONEOF(number, double) {
+        return js.num(number);
+      }
+    }
+    KJ_UNREACHABLE;
+  } else {
+    return js.null();
+  }
+}
+
+jsg::JsObject SqlStorage::wrapSqlRow(jsg::Lock& js, SqlRow row) {
+  return jsg::JsObject(js.withinHandleScope([&]() -> v8::Local<v8::Object> {
+    jsg::JsObject result = js.obj();
+    for (auto& field: row.fields) {
+      result.set(js, field.name, wrapSqlValue(js, kj::mv(field.value)));
+    }
+    return result;
+  }));
```

Comment:
```suggestion
  return js.withinHandleScope([&]() -> jsg::JsObject {
    jsg::JsObject result = js.obj();
    for (auto& field: row.fields) {
      result.set(js, field.name, wrapSqlValue(js, kj::mv(field.value)));
    }
    return result;
  });
```

Reply:
This doesn't work. `withinHandleScope` is not specialized for `jsg::JsObject` so it won't properly allow the handle to escape the scope.

Reply:
Ah right. Ok. Mind adding a comment?